rest-fights/src/main/java/io/quarkus/sample/superheroes/fight/service/FightService.java

```diff
@@ -210,4 +210,28 @@ Fight villainWonFight(Fighters fighters) {
 		fight.loserTeam = this.fightConfig.hero().teamName();
 		return fight;
 	}
+
+  @Timeout(value = 5, unit = ChronoUnit.SECONDS)
+  @Fallback(fallbackMethod = "fallbackHelloHeroes")
+  public Uni<String> helloHeroes() {
+    Log.debug("Invokes the Heroes microservice");
+    return this.heroClient.helloHeroes();
```

Comment:
Now that this is reactive, the log message should be in the reactive pipeline and not in the method itself. The execution of the method and the execution of the pipeline happen at 2 different points in time.

like

```java
return this.heroClient.helloHeroes()
  .invoke(hello -> Log.debugf("Got %s from the Heroes microservice", hello));
```